css/style.css

```diff
@@ -345,6 +345,28 @@ background-color: #FFF;
     }
 /*********Section-Sharpen your skills ends**********/
 
+/*********FAQ section starts**********/
+
+.fnq .card-header {
+    padding: .75rem 1.25rem;
```

Comment:
Except `box-shadow` rest all are default properties that comes from bootstrap itself so they can be removed.